package.json

```diff
@@ -50,8 +50,8 @@
     "release:publish": "scripts/release-publish.sh"
   },
   "dependencies": {
-    "algoliasearch-helper": "^2.26.1",
-    "instantsearch.js": "^3.6.0"
+    "algoliasearch-helper": "0.0.0-5a0352a",
```

Comment:
This can be using v3 now.

Reply:
InstantSearch hasn't yet been released, so was waiting for that